tests/Core/Components/Switch/FluentSwitchTests.razor

```diff
@@ -0,0 +1,88 @@
+﻿@using Microsoft.FluentUI.AspNetCore.Components.Extensions
+@using Microsoft.FluentUI.AspNetCore.Components.Tests.Extensions
+@using Microsoft.FluentUI.AspNetCore.Components.Utilities
+@using System.ComponentModel.DataAnnotations
+@using Xunit;
+@inherits TestContext
+
+@code
+{
+    public FluentSwitchTests()
+    {
+        JSInterop.Mode = JSRuntimeMode.Loose;
+        Services.AddFluentUIComponents();
+    }
+
+    [Fact]
+    public void FluentSwitch_Default()
+    {
+        // Arrange && Act
+        var cut = Render(@<FluentSwitch />);
+        // Assert
+        cut.Verify();
+    }
+
+    [Fact]
+    public void FluentCheckbox_LabelPosition()
+    {
+        // Arrange && Act
+        var cut = Render(
+            @<FluentSwitch LabelPosition="LabelPosition.Above" />
+        );
+
+        // Assert
+        cut.Verify();
+    }
```

Comment:
When an enum parameters is involved, we want to have tests for all possible values (including `null` and a random numeric value). You need to use [Theory] and [InlineData()] and `cut.Verify(suffix:...)`

See for an example the `FluentCounterBadge_Positioning` test.

Reply:
ok, Tests added with InlineData on LabelPosition.